### DISCUSSION TITLE
Instance Collision Shape Dimensions are Static

### DISCUSSION THREAD

**Discussion by CianFlint** [godotengine/godot-proposals#10327] [2D]
Proposal to make the dimensions of a collision shape non-static. It should be possible to adjust the dimensions of the shape of a 2d node instance without globally affecting the shape of all instances of the scene

**Comment by CianFlint** [comment]
Should this be made into a bug report

**Comment by AThousandShips** [comment]
This isn't static, you just need to make them unique, you are sharing the shape (which is a resource) with other instances of the same resource

**Comment by Calinou** [comment]
This is already what https://github.com/godotengine/godot-proposals/issues/4672 aims to address (but in a different manner).
